py/mpconfig.h

```diff
@@ -739,4 +739,8 @@ typedef double mp_float_t;
 #define MP_UNLIKELY(x) __builtin_expect((x), 0)
 #endif
 
+// Use to refer to extern structs of which actual type is not known.
+// For instance 'extern struct _dummy_t mp_sys_stdin_obj;'
+struct _dummy_t;
```

Comment:
I just figured it is not needed to define this dummy struct at all: `extern struct _mp_dummy_t mp_sys_stdin_obj;` is fine as-is and the compiler doesn't need to know more to be able to make a pointer from it.

We could still declare _mp_dummy_t in misc.h but it does not do anything, as in: compiler won't warn you if you make typeos like ```extern struct _mp_oops_dummy_t``` so in the end it only adds to maintainance and serves no purpose. Unless there are compilers which do not handle it but I doubt that?